clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/gcsClient/AppengineGcsRepositoryClient.groovy

```diff
@@ -50,7 +50,7 @@ class AppengineGcsRepositoryClient implements AppengineRepositoryClient {
 
     def fullPath = repositoryUrl.substring(gsPrefix.length())
     if (applicationDirectoryRoot) {
-      fullPath + '/' + applicationDirectoryRoot
+      fullPath = fullPath + '/' + applicationDirectoryRoot
```

Comment:
Could be:
`fullPath += '/' + applicationDirectoryRoot`

or:
`fullPath += "/$applicationDirectoryRoot"`